app/assets/javascripts/host_checkbox.js

```diff
@@ -193,7 +193,12 @@ function build_modal(element, url) {
 }
 
 function build_redirect(url) {
-  var url = url + "?" + $.param({host_ids: $.foremanSelectedHosts});
+  var is_multiple = $("#multiple-alert");
```

Comment:
the function `build_modal` does the same, 
so what do you think about extract this logic ?
something like this:
```javascript
function is_multiple() {
  return $("#multiple-alert").data(('multiple'));
}
function get_bulk_param() {
  return is_multiple() ? {search: $("#search").val()} : {host_ids: $.foremanSelectedHosts}
}
```